velox/exec/SortBuffer.cpp

```diff
@@ -259,15 +262,19 @@ void SortBuffer::ensureOutputFits() {
     return;
   }
 
-  if (!estimatedOutputRowSize_.has_value()) {
+  if (!estimatedOutputRowSize_.has_value() || spiller_ != nullptr ||
+      batchSize == 0) {
```

Comment:
batch size can't be zero and can you just add VELOX_CHECK at the entrance of this function? Thanks!

Reply:
`const vector_size_t batchSize =
      std::min<uint64_t>(numInputRows_ - numOutputRows_, maxOutputRows)`, The batchSize maybe 0 if we input an empty RowVector.